frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -310,33 +312,6 @@ pub trait BenchmarkingConfig {
 	const MAXIMUM_TARGETS: u32;
 }
 
-/// A fallback implementation that transitions the pallet to the emergency phase.
-pub struct NoFallback<T>(sp_std::marker::PhantomData<T>);
```

Comment:
@kianenigma this was used [here](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/src/lib.rs#L475). I guess we do not want to run onchain elections as fallback in production?

Reply:
please use `frame_election_provider_support::NoElection` (whatever the exact name of the equivalent was).